src/Java/Lesson5/CivilAviation.java

```diff
@@ -0,0 +1,43 @@
+package Java.Lesson5;
+
+import java.util.Scanner;
+
+public class CivilAviation extends AirTransport {
+    private static int passengers;
+    int numberOfPassengers;
+    boolean businessClassAvailability;
+
+    public void trafficAir() {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Введите количесиво пассажиров: ");
+        passengers = Integer.parseInt(in.nextLine());
+    }
+    public void brand() {
+        brand = "AirBus";
+        power = 12333;
+        maxSpeed = 12225;
+        weight = 884885;
+        wingspan = 350;
+        landingHeight = 100;
+        numberOfPassengers = 110;
+        businessClassAvailability = false;
```

Comment:
Замени на конструктор (не забудь про классы предки) чтобы можно было создавать объект по типу:
```suggestion
    public AirTransport(
          String brand,
          int power, 
          int maxSpeed, 
          int weight, 
          int wingspan, 
          int landingHeight, 
          int numberOfPassengers, 
          boolean businessClassAvailability 
    ) {
        this.brand = brand ;
        this.power = power;
        this.maxSpeed = maxSpeed;
        this.weight = weight;
        this.wingspan = wingspan;
        this.landingHeight = landingHeight;
        this.numberOfPassengers = numberOfPassengers ;
        this.businessClassAvailability = fabusinessClassAvailabilityse;
```
В итоге можно будет делать просто;
```suggestion
    AirTransport plane = new AirTranspot(
               "AirBus", 
               12333, 
               12225, 
               884885, 
               350, 
               100, 
               110,
               false
     )
```